e2e-tests/yarn.lock

```diff
@@ -53,6 +53,11 @@
   resolved "https://registry.npmjs.org/@humanwhocodes/object-schema/-/object-schema-2.0.3.tgz"
   integrity sha512-93zYdMES/c1D69yZiKDBj0V24vqNzB/koF26KPaagAfd3P/4gUlh3Dys5ogAK+Exi9QyzlD8x/08Zt7wIKcDcA==
 
+"@ioredis/commands@^1.1.1":
```

Comment:
Are you sure you want to commit the changes to this file?

Reply:
Yes, because I added a new package.

Reply:
@subhashkhileri  but I think this is generated automatically, you only need to change the package.json